create.sql

```diff
@@ -125,6 +125,7 @@ CREATE TABLE IF NOT EXISTS `maillog` (
   `headers` mediumtext COLLATE utf8_unicode_ci,
   `quarantined` tinyint(1) DEFAULT '0',
   `rblspamreport` mediumtext COLLATE utf8_unicode_ci DEFAULT NULL,
+  `token` mediumtext COLLATE utf8_unicode_ci DEFAULT NULL,
```

Comment:
`CHAR(40)` (or `CHAR(64)` for future sha256 support) should be better than mediumtext